samples/AlarmBot-Cards/Controllers/MessagesController.cs

```diff
@@ -22,27 +22,23 @@ namespace AlarmBot.Controllers
     [Route("api/[controller]")]
     public class MessagesController : Controller
     {
-        public static BotFrameworkAdapter activityAdapter = null;
-        public static Bot bot = null;
+        public static BotFrameworkBotServer botServer = null;
```

Comment:
Do we really need a public static member? For a different check in, might be worth revisiting the need for this pattern

Reply:
Who accesses this static member from outside of the controller?

Reply:
You want it to be a singleton, the controller is created on every request.